client/me/account/style.scss

```diff
@@ -24,6 +24,10 @@
 	width: 100%;
 }
 
+.account__settings-form .sites-dropdown .site-selector .search {
```

Comment:
Do we really need all the specificity? I think we can ditch either `.sites-dropdown` or `.site-selector` from this selector.

Reply:
I'm not sure but I don't think it's a bad thing, no? This rule specifically targets the search field living in the site selector in sites dropdown (there are others). If you don't feel strongly about it I'd just leave it like it is.

Reply:
I don't feel strongly about it 😉  It's just getting a bit too specific. I'm sure there was a document that describes the recommended max specificity, but I can't find it. @gwwar do you have it at the top of your head?

Reply:
No hard rules, but Calypso was using a form of BEM so in the docs, so specificity usually doesn't go past (0,0,2). eg `.component__element.is-modifier` https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md#example

If y'all wanted to refactor, it'd probably be giving a specific component name to search.

Reply:
Thanks @gwwar 🙌